.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I think that file should also be added in the following locations as we often open these projects separately in VSCode:
- Website/.vscode
- Website/src/rendering/.vscode
- demo/experience/cdp/guestDataGenerator/.vscode
- kiosk/.vscode
- tv/.vscode

Then we should add an entry in the `compare-synced-files.ps1` file comparison script to ensure all the copies of the file are in sync.

Thanks!

Reply:
Made changes as requested